tools/main.py

```diff
@@ -2,6 +2,7 @@
 import os.path as osp
 import sys
 import time
+from pprint import pformat
```

Comment:
Seems like `pformat` is not used in this file

Reply:
Fixed